Resources/Prototypes/Entities/Clothing/Head/hardsuit-helmets.yml

```diff
@@ -751,3 +751,30 @@
     color: "#f4ffad"
     radius: 5
     energy: 2
+
+#Debug Hardsuit
+- type: entity
+  parent: ClothingHeadHelmetHardsuitSyndieCommander
+  id: ClothingHeadHelmetHardsuitAdmin
+  noSpawn: true
+  name: bluespace traveler hardsuit helmet [admeme]
```

Comment:
Why is it nospawn and why is [admeme] here and not a suffix for admins

Reply:
arent all hardsuit helmets meant to be nospawn since they are spawned by the hardsuit

Reply:
Corrected